raiden/api/v1/resources.py

```diff
@@ -151,6 +151,14 @@ def get(self, **kwargs):
         )
 
 
+class UnlockedPaymentLightResource(BaseResource):
+    post_schema = UnlockedPaymentLightPutSchema()
+
+    @use_kwargs(post_schema)
+    def post(self, signed_tx: typing.SignedTransaction, **kwargs):
+        return self.rest_api.post_unlocked_payment_light(signed_tx, **kwargs)
```

Comment:
it's weird that we are passing keyword arguments to the `rest_api` method, when on the other side it expects _only_ a signed transaction plus a token address (and nothing else).

but this problem is too big to be fixed in this PR, so i'd leave as is.